packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -147,7 +146,7 @@
                 x-transition:leave="duration-300"
                 @if ($width === MaxWidth::Screen)
                 @elseif ($slideOver)
-                    x-transition:enter-start="translate-x-full rtl:-translate-x-full"
+                    x-transition:enter-start="translate-x-2/3 rtl:-translate-x-2/3"
```

Comment:
Could you elaborate on how this fixes the issue, @andrewdwallo?
Also, does adding `will-change` here make any difference?

Reply:
Also, even if `will-change: transform` was applied to the correct element, there would most likely be no benefit. I tested it myself and saw no improvement, so it can depend, but it is best to not use it at all.